### PR TITLE
Added feature - Support to generate multiple images with single click

### PR DESCRIPTION
**Pull request by anirudh1200**
Feature Added:
- Text bigger than a page can be added and converted to images in one click and multiple pages will be generated automatically
- Added navigation to view all the generated pages

handwriting

### CODE REVIEW

**Comment by saurabhdaware** [comment]
Hi @anirudh1200, I went through the files changes and it looks great so far! I am a little busy this week so I will properly test the PR by the next weekend. Thank You!

File: index.html [hunk 1]
```diff
@@ -171,6 +171,11 @@ <h1 style="margin-bottom: 7px;">Text to Handwriting</h1>
         </div>
         <div class="col output-container">
           <div id="output" class="output"></div>
+          <div style="text-align: center;padding-top: 20px;display: none;" id="nav">
+            <span class="blue-button" id="page-nav-left">&#60;&#8722;</span>
```
Comment (saurabhdaware) [inline index.html]: Will it break anything if we turn it into `<button>`? Currently, these buttons would be inaccessible to the people using the keyboard tab to navigate

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
This is absolutely amazing @anirudh1200! Thank you so much and everything works great 🎉 I've just requested a small accessibility change and we can get this merged 🕺🏻 Thank you so much!

File: index.html (hunk 1 above)
Comment (anirudh1200) [reply index.html]: No, it will not break anything on the site. I have made the changes and tested it and it works fine !

**Review by saurabhdaware — APPROVED** [review]
Awesome! Thank you so much 🌻

This PR is super cool! I don't have lots of Twitter followers but Is there anything (Twitter account, Any project, or other accounts) that you would like me to give you a shoutout for on Twitter?

**Comment by anirudh1200** [comment]
Wow dude thanks a lot for your offer !!!

No dude, but currently i do not have a twitter account.

**Comment by saurabhdaware** [comment]
No problem and thank you for the PR 🥳

**Pull request merged**